systemtest/src/test/java/io/strimzi/systemtest/kafka/KafkaST.java

```diff
@@ -804,6 +422,18 @@ void testTopicWithoutLabels(ExtensionContext extensionContext) {
         assertThat(topics, not(hasItems("topic-without-labels")));
     }
 
+    /**
+     * @description This test case verifies that Kafka with persistent storage, and JBOD storage, PVCs set to delete true in one and false in another..
+     *
+     * @steps
```

Comment:
it looks you are missing steps for `verifyVolumeNamesAndLabels`, `clusterDeletion` and `waitingForDeletion`